### ISSUE TITLE
Link to documentation is broken

### ISSUE THREAD

**Issue by flapflap** [obsidian-community/obsidian-full-calendar#576]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/davish/obsidian-full-calendar/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and didn't find any similar reports.

### Expected Behavior

Link is working

### Current behaviour

Link (https://davish.github.io/obsidian-full-calendar/) provide an Error 404

### Steps to reproduce

Go to the main page, click on the full documentation link
https://github.com/obsidian-community/obsidian-full-calendar?tab=readme-ov-file

### Calendar information

NR

### Which Operating Systems are you using?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] Android
- [ ] iOS

### Obsidian Version

NR

### Full Calendar Plugin Version

NR

### Checks

- [ ] I have tried it with all other plugins disabled and the error still occurs.

**Comment by chapost1** [comment]
same

**Comment by Riven-nen** [comment]
same here :(

**Comment by winged-voices** [comment]
yep :C this one's under obsidian-community -- should that be where it redirects to?

**Comment by geogrrl** [comment]
> yep :C this one's under obsidian-community -- should that be where it redirects to?
> 
> [Edit: That fixes it! Link should be: https://obsidian-community.github.io/obsidian-full-calendar/ . Just need someone with admin access to fix the link in the sidebar/at the bottom of the README as well I think.]

yeah this was really annoying. barely anyone maintains stuff. thanks for the updated link!

**Comment by etu51424** [comment]
Not really a solution but the documentation can be found here as well: http://web.archive.org/web/20231127165354/https://davish.github.io/obsidian-full-calendar/
